<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResolutionIdentifier</class>
 <widget class="QDialog" name="ResolutionIdentifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGraphicsView" name="graphicsView_2"/>
   </item>
   <item row="0" column="0">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="1" column="0">
    <widget class="QGraphicsView" name="graphicsView_3"/>
   </item>
   <item row="1" column="1">
    <widget class="QGraphicsView" name="graphicsView_4"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
